src/main/scala/beam/agentsim/agents/freight/input/GenericFreightReader.scala

```diff
@@ -183,15 +183,19 @@ class GenericFreightReader(
       case Some(filePath) => vehicleTypes ++ readBeamVehicleTypeFile(filePath)
       case None           => vehicleTypes
     }
-
-    val existingTours: Set[Id[FreightTour]] = allTours.keySet.intersect(allPlans.map(_._2.tourId).toSet)
+    // ****
```

Comment:
I suggest adding a comment or removing it.

Reply:
I prefer your suggestion. I committed your suggestion.